packages/core/src/transport/transport.ts

```diff
@@ -29,7 +51,41 @@ export class HttpRequest {
         reportBeaconError(e)
       }
     }
+
+    const transportIntrospection = (event: ProgressEvent) => {
+      const req = event?.currentTarget as XMLHttpRequest
+      if (req.status >= 200 && req.status < 300) {
+        return
+      }
+      const xhrTransportContext: TransportIssueContext = {
+        on_line: navigator.onLine,
+        size,
+        url,
+        try_beacon: tryBeacon,
+        bytes_limit: this.bytesLimit,
+        event: {
+          type: event.type,
+          is_trusted: event.isTrusted,
+          total: event.total,
+          loaded: event.loaded,
+        },
+        request: {
+          status: req.status,
+          ready_state: req.readyState,
+          response_text: req.responseText,
```

Comment:
I'd be in favor of truncating the response to avoid large payloads

Reply:
`req.responseText.slice(0, 64)`